src/components/ZetkinCalendar/MonthCalendar/index.tsx

```diff
@@ -55,7 +66,7 @@ const MonthCalendar = ({ orgId, campaigns, baseHref, events, focusDate, tasks }:
 
     const firstMonthDay = new Date(year, month, 1);
     const lastMonthDay = new Date(year, month, totalDaysInMonth + 1);
-    const firstCalendarDay = new Date(new Date(firstMonthDay).setDate(firstMonthDay.getDate() - (firstMonthDay.getDay() || 7) + 1 ));
+    const { firstDayInView } = getViewRange(focusDate, CALENDAR_RANGES.MONTH);
```

Comment:
Curious to hear your thoughts on this @nehahirve

Reply:
it looks good, i think i'd have done similar